docs/HowToFAQ/Errors-Compiler.md

```diff
@@ -0,0 +1,104 @@
+<!-- DafnyCore/Compilers/SinglePassCompiler.cs -->
+
+## {0} Process exited with exit code {1}
+
+
+## Error: Unable to start {psi.FileName}{additionalInfo}
+
+## Compilation error: _msg_
+
+## '_feature_' is not an element of the {TargetId} compiler's UnsupportedFeatures set
+
+## Opaque type ('{0}') with extern attribute requires a compile hint.  Expected {{:extern compile_type_hint}} 
+
+## Opaque type ('{0}') cannot be compiled; perhaps make it a type synonym or use :extern.
+
+## since yield parameters are initialized arbitrarily, iterators are forbidden by the --enforce-determinism option
```

Comment:
I am nitpicking: it would be nice to have a consistent sentence style here and elsewhere (e.g. start with a uppercase/lowercase letter and end with/without a full-stop).

Reply:
I agree. The bulk of error messages begin with a lower-case letter and end without a period. Some, particularly where there are multiple sentences begin with upper-case and end with a period. I would like to make it more consistent, but I'd like to leave that to a separate PR, sine it will require major changes to Dafny.atg and to many tests.

Reply:
Sounds good